src/components/AudioPlayer/AudioPlayer.module.scss

```diff
@@ -31,5 +31,5 @@
 }
 
 .containerMinimized.containerDefault {
-  height: constants.$audio-player-minimized-height;
+  transform: translate3d(0, constants.$audio-player-minimized-height, 0);
```

Comment:
based on some resource, it says translate3d is faster than `translate`.
but I will double check

Reply:
Nice, we should add this comment in the code